src/sentry/static/sentry/app/views/releaseDetails.jsx

```diff
@@ -18,6 +21,54 @@ import TimeSince from 'app/components/timeSince';
 import Version from 'app/components/version';
 import withEnvironmentInQueryString from 'app/utils/withEnvironmentInQueryString';
 
+class ReleaseDetailsActions extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    orgId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+    release: PropTypes.object.isRequired,
+  };
+
+  handleDelete = () => {
+    let {orgId, projectId, release, api} = this.props;
+    let version = encodeURIComponent(release.version);
+    let path = `/organizations/${orgId}/releases/${version}/`;
+    let redirectPath = `/${orgId}/${projectId}/releases/`;
+    let indicator = IndicatorStore.add(t('Deleting Release...'));
+
+    api.request(path, {
+      method: 'DELETE',
+      success: () => {
+        IndicatorStore.remove(indicator);
+        browserHistory.push(redirectPath);
+      },
+      error: err => {
+        IndicatorStore.remove(indicator);
+        IndicatorStore.addError(t(err.responseJSON.detail));
+      },
+    });
+  };
+
+  render() {
+    return (
+      <div className="release-actions m-b-1">
+        <div className="btn-group">
+          <LinkWithConfirmation
+            className="release-remove btn btn-default btn-sm"
```

Comment:
You'll notice we still use these bootstrap class names for buttons, the preferred method is to use the `app/components/button` component except in this case where:

1) `btn-group` is used (we don't have an equivalent react component for button groups yet), and
2) ~~`LinkWithConfirmation`~~ (read below) and other components that don't accept a generic component to render.

Reply:
I was able to use `Confirm` and `Button` to get what I wanted.